### PR TITLE
Sending to queue

### PR DESCRIPTION
**Pull request by rychuhardy**

### CODE REVIEW

File: src/Server/src/main/java/com.parking/controller/SampleController.java [hunk 1]
```diff
@@ -11,8 +14,13 @@
 @RestController
 public class SampleController {
```
Comment (karolkozak) [inline src/Server/src/main/java/com.parking/controller/SampleController.java]: Rename to ImageController or something

File: src/Server/src/main/java/com.parking/controller/SampleController.java [hunk 2]
```diff
@@ -11,8 +14,13 @@
...
 
+    @Autowired
+    private IQueueMessageSender sender;
+
     @RequestMapping(method = RequestMethod.GET)
     public ResponseEntity<ApiResponseBody> getSampleResponse() {
+         ImageMessage msg = new ImageMessage("abc", "test bytes".getBytes());
+         boolean succeeded = sender.send(msg);
         return new ResponseEntity<>(new ApiResponseBody("SampleResponse"), HttpStatus.OK);
```
Comment (karolkozak) [inline src/Server/src/main/java/com.parking/controller/SampleController.java]: HttpStatus.PROCESSING and adjust response message.

File: src/Server/src/main/java/com.parking/model/ImageMessage.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+import java.io.Serializable;
+
+@Data
+@AllArgsConstructor
+public class ImageMessage implements Serializable {
+    private String message;
```
Comment (karolkozak) [inline src/Server/src/main/java/com.parking/model/ImageMessage.java]: Maybe integer, as ID from database?

File: src/Server/src/main/resources/application.yml [hunk 4]
```diff
@@ -7,3 +7,7 @@ cors:
     access:
         allowed:
             origins: http://localhost:4200
+azure:
+    servicebus:
+        connection-string: Endpoint=sb://parkingapp.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=pvmiqJ5A1UJP4shCNasRC1iC8Wng7AdUv17kf5ZiTfI=
+        queue-name: plates
```
Comment (karolkozak) [inline src/Server/src/main/resources/application.yml:13]: Eee, plates?

**Review by karolkozak — CHANGES_REQUESTED** [review]
Your changes have not built, see:
https://travis-ci.org/karolkozak/Parking-Service/builds/372955745?utm_source=github_status&utm_medium=notification

File: src/Server/src/main/java/com.parking/controller/SampleController.java (hunk 2 above)
Comment (rychuhardy) [reply src/Server/src/main/java/com.parking/controller/SampleController.java]: Well, I thought that you will implement an endpoint  for uploading images, so this is just a sample usage. Anyway, in the end the method should be POST and the response 202 is more appropriate.

Comment (karolkozak) [reply src/Server/src/main/java/com.parking/controller/SampleController.java]: ok

**Review by karolkozak — APPROVED** [review]

**Pull request merged**